Tribute Page/index.html

```diff
@@ -15,50 +15,62 @@ <h3>
+          <i>Musk</i> attended the English-speaking Waterkloof House Preparatory School, and later graduated from Pretoria Boys High School.<br/> At 17, Musk moved to Canada to avoid serving in the South African military, whose main duty in the late 1980s was
           enforcing apartheid. SAfter emigrating to Canada, Musk enrolled in Queen's University in Kingston, Ontario. He studied physics there. <br/> Also he graduated with bachelor's degree in economics from the Wharton school of business.<br/> At 24,
-          he went to pursue his PhD. in applied Physics at Stanford University.
+          he went to pursue his PhD. in applied Physics at <a href="https://www.stanford.edu">Stanford University</a>.
         </p>
       </div>
       <div id="boe">
         <h3>
-        The birth of an Entrepeneur
+        </i><strong>The birth of an Entrepeneur</i></strong>
       </h3>
         <p>
-          Just after two days being enrolled into Stanford University, he dropped out to start his own comapany <strong>Zip2</strong> which was an online city guide. It also provided content for new websites of both The New York Times and the Chicago
-          Tribune, and in 1999, a division of Compaq Computer Corporation bought Zip2 for $307 million in cash and $34 million in stock options.
+          Just after two days being enrolled into Stanford University, he dropped out to start his own comapany <strong><i>
+          <a href="https://en.m.wikipedia.org/wiki/Zip2">Zip2 </a></i></strong> 
+          which was an online city guide. It also provided content for new websites of both The New York Times and the Chicago
+          Tribune, and in 1999, a division of Compaq Computer Corporation bought Zip2 for
+          $307 million in cash and $34 million in stock options.
         </p>
         <p>
-          Also, in 1999 Musk co-founded X.com, an online financial services/payments company. An X.com acquisition the following year led to the creation of PayPal as it is known today, and in October 2002, PayPal was acquired by eBay for $1.5 billion in stock.
+          Also, in 1999 Musk co-founded <strong><i><a href="https://en.m.wikipedia.org/wiki/X.com">X.com</a></i></strong>, an online financial services/payments company.
+          An X.com acquisition the following year led to the creation of <strong><i><a href="https://www.paypal.com">PayPal</a></i></strong> as it is known today, and in October 2002,
+          PayPal was acquired by eBay for $1.5 billion in stock.
           Before the sale, Musk owned 11 percent of PayPal stock.
         </p>
       </div>
       <div id="change">
         <h3>
-        Changing Today for a better Tomorrow
+        </i><strong>Changing Today for a better Tomorrow</i></strong>

```

Comment:
```suggestion
        <i><strong>Changing Today for a better Tomorrow</strong></i>
```
Please fix this issue by placing the tags correctly in addition to fixing the opening <i> tag.